release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -85,6 +85,15 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-nodes"]
 === Nodes
 
+[id="ocp-4-8-nodes-descheduler-metrics"]
+==== Prometheus metrics for the descheduler
+
+You can now enable Prometheus metrics for the descheduler by adding the `openshift.io/cluster-monitoring=true` label to the `openshift-kube-descheduler-operator` namespace where you installed the descheduler.
+
+The following descheduler metrics are available:
+
+* `descheduler_build_info` - Provides build information about the descheduler.
```

Comment:
@bergerhoffer not sure if the description for this is apt, as below is the info we see on prometheus for the descheduler_build_info, which does not contain anything about descheduler build information.

**descheduler_build_info{GoVersion="go1.16.1", endpoint="https", instance="10.129.3.143:10258", job="metrics", namespace="openshift-kube-descheduler-operator", pod="cluster-6c89d56fdd-5qzrq", service="metrics"}**

Reply:
It depends on how the descheduler operand was actually built. Checking https://github.com/openshift/descheduler/blob/master/images/descheduler/Dockerfile.rhel7#L4 we don't set the right LDFLAGS. Lemme fix that. Though, the description is still valid.

Reply:
https://github.com/openshift/descheduler/pull/59

@kasturinarra thanks for noticing that.